lib/components/fileListItem/fileListItem.js

```diff
@@ -22,8 +31,34 @@ const FileListItem = ({ name, extension, size, tooltipText, warning, button, dec
     </>
   );
 
+  const keyDown = (event) => {
+    const { keyCode, target } = event;
+
+    // eslint-disable-next-line no-magic-numbers
+    if (keyCode === 37 || keyCode === 39) {
```

Comment:
I agree with ESLint here, what are these numbers? Can you make constants for them?